bookmarks/account/views.py

```diff
@@ -70,3 +73,18 @@ def edit(request):
         "account/edit.html",
         {"user_form": user_form, "profile_form": profile_form},
     )
+
+
+@ajax_required
+@require_POST
+@login_required
+def user_follow(request):
```

Comment:
follow_user_view

But even then the name is misleading since the view does two things follow and unfollow. This violation of SRP.

Either we need to have a separate endpoint for follow and unfollow or the view name should be explicitly clear about it doing both actions. For this demo project we could stick with the later suggestion of explicit naming of view.